SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -168,6 +168,11 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(config)
 	bool bEnableServerQBI;
 
+	/** EXPERIMENTAL - Adds granular result types for client queries.
+	Needs testing thoroughly before making default. May be replaced by component set result types instead. */
+	UPROPERTY(config)
+	bool bEnableGranularResultTypes;
```

Comment:
Nit: This variable doesn't reflect it's being used for clients. Potentially add Client to it?

Reply:
I was planning to put all the result type work behind this flag- follow up work includes setting result types for client self-constraints and potentially server workers, however if you think there should be separate flags for only client checking out other entities happy to rename

Reply:
change granular->client